src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -342,9 +342,15 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
 
     Windows::UI::Xaml::Automation::Peers::AutomationPeer TermControl::OnCreateAutomationPeer()
     {
-        // create a custom automation peer with this code pattern:
-        // (https://docs.microsoft.com/en-us/windows/uwp/design/accessibility/custom-automation-peers)
-        return winrt::make<winrt::Microsoft::Terminal::TerminalControl::implementation::TermControlAutomationPeer>(*this);
+        Windows::UI::Xaml::Automation::Peers::AutomationPeer autoPeer = nullptr;

```

Comment:
prefer universal initialization

```suggestion
        Windows::UI::Xaml::Automation::Peers::AutomationPeer autoPeer{ nullptr };
```